Editor/CodeAnalysis/CallTreeNode.cs

```diff
@@ -56,6 +57,8 @@ internal CallTreeNode(MethodReference methodReference, CallTreeNode caller = nul
                 methodName = methodReference.Name;
             }
 
+            declaringType = methodReference.DeclaringType.ToString();
```

Comment:
@borisbauer-unity do you remember the reason for adding this? I think this is the same as `typeName`. We do some special parsing to make coroutine names more user-friendly but other than that, `declaringType `should end up being the same as `declaringType`. What do you think?